packages/eslint-config-airbnb-base/rules/style.js

```diff
@@ -3,531 +3,823 @@ module.exports = {
-    'array-bracket-newline': ['off', 'consistent'], // object option alternative: { multiline: true, minItems: 3 }
+    // KEEP AS IS
+    "array-bracket-newline": ["off", "consistent"], // object option alternative: { multiline: true, minItems: 3 }
 
     // enforce line breaks between array elements
     // https://eslint.org/docs/rules/array-element-newline
     // TODO: enable? semver-major
-    'array-element-newline': ['off', { multiline: true, minItems: 3 }],
+    // KEEP, SITUATIONAL ANALYSIS NEEDED
+    // Need to determine how many items to allow on single line before breaking out to 1 per line
+    "array-element-newline": ["off", { multiline: true, minItems: 3 }],
 
     // enforce spacing inside array brackets
-    'array-bracket-spacing': ['error', 'never'],
+    // KEEP
+    "array-bracket-spacing": ["error", "never"],
 
     // enforce spacing inside single-line blocks
     // https://eslint.org/docs/rules/block-spacing
-    'block-spacing': ['error', 'always'],
+    // DISABLE - can be replaced by Prettier's configuration: bracketSpacing: true
+    "block-spacing": ["error", "always"],
 
     // enforce one true brace style
-    'brace-style': ['error', '1tbs', { allowSingleLine: true }],
+    // KEEP
+    "brace-style": ["error", "1tbs", { allowSingleLine: true }],
 
     // require camel case names
-    camelcase: ['error', { properties: 'never', ignoreDestructuring: false }],
+    // KEEP, would prefer to keep as 'error' to disallow auto-fix. Force the developer to manually choose a proper variable name
+    camelcase: ["error", { properties: "never", ignoreDestructuring: false }],
 
     // enforce or disallow capitalization of the first letter of a comment
     // https://eslint.org/docs/rules/capitalized-comments
-    'capitalized-comments': ['off', 'never', {
-      line: {
-        ignorePattern: '.*',
-        ignoreInlineComments: true,
-        ignoreConsecutiveComments: true,
-      },
-      block: {
-        ignorePattern: '.*',
-        ignoreInlineComments: true,
-        ignoreConsecutiveComments: true,
+    // DISABLE - I'm not particularly bothered one way or the other on comment capitalization.
+    "capitalized-comments": [
+      "off",
+      "never",
+      {
+        line: {
+          ignorePattern: ".*",
+          ignoreInlineComments: true,
+          ignoreConsecutiveComments: true,
+        },
+        block: {
+          ignorePattern: ".*",
+          ignoreInlineComments: true,
+          ignoreConsecutiveComments: true,
+        },
       },
-    }],
+    ],
 
     // require trailing commas in multiline object literals
-    'comma-dangle': ['error', {
-      arrays: 'always-multiline',
-      objects: 'always-multiline',
-      imports: 'always-multiline',
-      exports: 'always-multiline',
-      functions: 'always-multiline',
-    }],
+    // DISABLE - I'm very anti-comma-dangling
+    // Can be managed by Prettier configuration: trailingComma: 'none'
+    "comma-dangle": [
+      "error",
```

Comment:
Turn this to `off` - but ESLint Prettier Recommended will likely disable this.

Reply:
turned off